lib/auth/session_access_test.go

```diff
@@ -164,7 +164,26 @@ func failFilterStartTestCase(t *testing.T) startTestCase {
 				Mode:     "peer",
 			},
 		},
-		expected: false,
+		expected: []bool{false},
+	}
+}
+
+func versionFallbackStartTestCase(t *testing.T) startTestCase {
+	hostRole, err := types.NewRoleV3("host", types.RoleSpecV5{})
```

Comment:
Why are we using `NewRoleV3` here and in the `versionDefaultJoinTestCase`?
I would expect a call to `NewRole`

Reply:
Ah, the reason we're calling this here, is that this testcase makes sure that a participant with no v5 roles retains the same privileges as pre-moderated-sessions join RBAC tightening.